BUILD

```diff
@@ -226,6 +227,7 @@ cc_library(
     "src/core/json/json_common.h",
     "src/core/json/json_reader.h",
     "src/core/json/json_writer.h",
+    "src/core/proto/load_balancer.pb.h",
```

Comment:
The C++ output is used in the grpclb_api_test file to encode/decode the test protos used, for convenience (the C api doesn't have all the encoding/decoding functions, only the ones needed by the grpclb policy).